### PR TITLE
[bitnami/moodle] Allow to define default site lang with environment variable

### PR DESCRIPTION
**Pull request by rimkov**
Signed-off-by: Karim Ayari <karim.ayari@univ-lyon1.fr>

### Description of the change

Allow to define default Moodle site lang with environment variable

### Benefits

Avoids additional manipulations after installation

### Additional information

the language will be passed to the install.php command and will therefore allow the installer to automatically download the associated pack

### Checklist

- [x] Variables are documented in the values.yaml and added to the `README.md` using [readme-generator-for-helm](https://github.com/bitnami-labs/readme-generator-for-helm)
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

**Comment by carrodher** [comment]
Could you please take a look at the failing actions? You need to bump the chart version and add the metadata for the recently added parameter in the values.yaml; then the entry in the README.md will be automatically added

File: bitnami/moodle/values.yaml [hunk 1]
```diff
@@ -81,6 +81,9 @@ moodleSkipInstall: false
 ## ref: https://github.com/bitnami/containers/tree/main/bitnami/moodle#configuration
 ##
 moodleSiteName: ""
+## @param moodleLang Site language
+## ref: https://github.com/bitnami/containers/tree/main/bitnami/moodle#configuration
+moodleLang: ""
```
Comment (carrodher) [inline bitnami/moodle/values.yaml:86]:
```suggestion
## @param moodleLang Site language
## ref: https://github.com/bitnami/containers/tree/main/bitnami/moodle#configuration
##
moodleLang: ""
```

**Review by carrodher — COMMENTED** [review]
Thanks! It looks better now, just a couple of minor comments...
- Can you please bump the chart version in a minor since this PR is adding a new feature instead of a bug fix?
- See the inline suggestion (you can directly apply it)

File: bitnami/moodle/values.yaml (hunk 1 above)
Comment (rimkov) [reply bitnami/moodle/values.yaml:86]: indeed I have just seen that the chat is already in version 14.2.6. I will bump it in 14.2.7

**Review by carrodher — APPROVED** [review]

**Comment by rimkov** [comment]
thank you for your help :)

**Pull request merged**